downloads/tests/settings/19.py

```diff
@@ -0,0 +1,97 @@
+import os
+from os.path import expanduser
+
+SECRET_KEY = "PLACE_SECRET_KEY_HERE"
+
+USE_TZ = True
+
+TIME_ZONE = "Africa/Johannesburg"
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'mydatabase'
+    }
+}
+
+INSTALLED_APPS = (
+    "downloads",
+    "test_without_migrations",
+    "jmbo",
+    "photologue",
+    "category",
+    "ckeditor",
+    "django_comments",
+    "likes",
+    "secretballot",
+    "pagination",
+    "preferences",
+    "ultracache",
+    "sites_groups",
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.sites",
+    "layers"
+)
+
+ROOT_URLCONF = "downloads.tests.urls"
+
+MIDDLEWARE_CLASSES = (
+    "django.middleware.common.CommonMiddleware",
+    "django.contrib.sessions.middleware.SessionMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    "likes.middleware.SecretBallotUserIpUseragentMiddleware",
+    "pagination.middleware.PaginationMiddleware",
+)
+
+TEMPLATE_CONTEXT_PROCESSORS = [
+    "django.contrib.auth.context_processors.auth",
+    "django.template.context_processors.debug",
+    "django.template.context_processors.i18n",
+    "django.template.context_processors.media",
+    "django.template.context_processors.static",
+    "django.template.context_processors.tz",
+    "django.template.context_processors.request",
+    "django.contrib.messages.context_processors.messages",
+]
+
+TEMPLATES = [
```

Comment:
If you add `downloads.tests` to INSTALLED_APPS then you can remove this entire TEMPLATES setting. That is because Django will see tests as an actual app and the app directories template loader will kick in.